packages/design-system/src/components/SimpleMenu/Menu.tsx

```diff
@@ -180,8 +182,19 @@ const MenuItem = ({ onSelect, disabled = false, isLink, ...props }: ItemProps) =
           <Typography>{props.children}</Typography>
         </OptionLink>
       ) : (
-        <OptionButton cursor="pointer" color="neutral800" background="transparent" borderStyle="none" {...props}>
-          <Typography>{props.children}</Typography>
+        <OptionButton
+          cursor="pointer"
+          color="neutral800"
+          background="transparent"
+          borderStyle="none"
+          gap={2}
+          {...props}
+        >
+          {props.startIcon}
```

Comment:
By doing this, you are authorising everything to be rendered. I won't ask you to add strong validation on the props but since we are assuming that user should only render non-interactive elements (in this use case, icons). I'd like to see a forced aria-hidden here. Same for the endIcon.

```suggestion
          {startIcon && <Box aria-hidden>{startIcon}</Box>}
```